integr8ly/integreatly-operator
# update fuse online version to 7.6

- Update Fuse Online to 7.6
- Update Fuse Online product reconciler to use external resources (postgres) provided by CRO
- Move alerts/grafana dashboards for Fuse Online into the intly operator so that postgres alerts and dashboards aren't created (they are not required when using external resources)
- Update cluster role permissions to allow for alerts/service monitors for Fuse Online to be created

Reviews:
- https://github.com/integr8ly/integreatly-operator/pull/597/commits/292a9db2bd3ef05ed6796230386c29092f789f7e - main code changes
- https://github.com/integr8ly/integreatly-operator/pull/597/commits/f8bb9bda26debc3a43af180b025cd06937e26db3 - functional testing changes
- https://github.com/integr8ly/integreatly-operator/pull/597/commits/1506418a56a0fcfd26c5cb49bf05df61d711a991 - monitoring/alerting changes

Verification:
- Install against an OSD4 cluster with `useClusterStorage` set to `false`
- Ensure the Fuse Online namespace does not contain a database pod
- Ensure Fuse Online is now version 7.6 (through the UI or pod image tags)
- Ensure a new RDS instance is created in AWS for Fuse Online
- Create an integration in Fuse by following the Fuse Online walkthrough

@jhellar @psturc Would you mind taking a look at the verification side of this? I'm happy to provision this against a BYOC cluster first if required.
